README.md

```diff
@@ -50,13 +50,19 @@ For now, we don't have many plugins to offer you, but follow us and in the next
 
 Plugin to manage terraform files, by default it comes with aws provider definition.
 
-[GitHub url](https://github.com/ditrit/terrator-plugin#0.8.1)
+[GitHub url](https://github.com/ditrit/terrator-plugin#0.10.0)
 
 ### Github plugin
 
-Plugin to manage github action files.
+Plugin to manage GitHub action files.
 
-[GitHub url](https://github.com/ditrit/githubator-plugin#0.3.3)
+[GitHub url](https://github.com/ditrit/githubator-plugin#0.4.1)
+
+### Kubernetes plugin
+
+Plugin to manage Kubernetes files.
+
+[GitHub url](https://github.com/ditrit/githubator-plugin#0.2.0)
```

Comment:
Shouldn't  it be the url to kubernator plugin instead of githubator?

Reply:
No because it's the project github url

Reply:
Sorry I missunderstand, repush ;)